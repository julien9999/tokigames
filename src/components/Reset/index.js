import React from 'react';
import { useDispatch } from 'react-redux';
import { setOrder } from '../../actions/orderActions';
import { setFilter } from '../../actions/filterActions';
import { SHOW_ALL_FILTER } from '../../constants/actionTypes';

export default function Reset() {
  const dispatch = useDispatch();

  return (
    <div className='container margin-top-twenty'>
        <div className='row'>
          <div className='col-12'>       
            <ul class="nav nav-pills nav-fill">
              <li className="nav-item">
                  <a onClick={() => {
                      dispatch(setFilter(SHOW_ALL_FILTER))
                      dispatch(setOrder(''))
                    }} 
                    className={`nav-link active red`}>
                      Reset
                  </a>
              </li>
            </ul>
          </div>
        </div>
    </div>
  );
}

