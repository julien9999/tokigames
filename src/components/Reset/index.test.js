import React from 'react';
import renderer from 'react-test-renderer';
import Reset from './index';
import { Provider } from 'react-redux';
import configure from '../../store/configureStore';

const store = configure();

describe('<Reset>', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
            <Reset />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

