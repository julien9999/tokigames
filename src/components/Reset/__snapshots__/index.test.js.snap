// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reset> renders correctly 1`] = `
<div
  className="container margin-top-twenty"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <ul
        class="nav nav-pills nav-fill"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link active red"
            onClick={[Function]}
          >
            Reset
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
